src/JHipsterNetSampleApplication/Models/RelationshipTools/JoinListFacade.cs

```diff
@@ -0,0 +1,83 @@
+using JHipsterNetSampleApplication.Models.Interfaces;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace JHipsterNetSampleApplication.Models.RelationshipTools {
+    public class JoinListFacade<TEntity, TOtherEntity, TJoinEntity>
+    : IList<TEntity>
+    where TJoinEntity : IJoinEntity<TEntity>, IJoinEntity<TOtherEntity>, new() {
+        private readonly TOtherEntity _ownerEntity;
+        private readonly IList<TJoinEntity> _list;
+
+        public JoinListFacade(
+            TOtherEntity ownerEntity,
+            IList<TJoinEntity> list)
+        {
+            _ownerEntity = ownerEntity;
+            _list = list;
+        }
+
+        public IEnumerator<TEntity> GetEnumerator()
```

Comment:
if you use `=>` the statement should be on the same line otherwise use `{ ... }`